tidepool-analysis-tools/tests/unit_tests/test_clean.py

```diff
@@ -0,0 +1,40 @@
+
+from tidals.clean.clean import remove_duplicates, round_time
+import pandas as pd
+from pandas.util import testing as tm
+import pytest
+from tests.fixtures.clean_setup import valid_df
+
+
+def test_remove_duplicates(valid_df):
+
+    duplicated_data =  [["jjety", "AB9960", "response1", "response2", "Sun Nov 19 23:59:59 2018", pd.to_datetime("2018-11-20 00:00:00")],
```

Comment:
@jameno and I started a code guideline doc that we should really post in this repository so that contributors can follow our code conventions. I will find the doc and share it with you, and would very much like your input regarding the conventions that we should follow.